src/viam/app/data_client.py

```diff
@@ -194,11 +194,12 @@ async def tabular_data_by_filter(
                 data.
             limit (int): The maximum number of entries to include in a page. Defaults to 50 if unspecified.
             sort_order (viam.proto.app.data.Order): The desired sort order of the data.
-            last (str): Optional string indicating the ID of the last-returned data.
-                        If provided, the server will return the next data entries after the `last` ID.
+            last (str): Optional string indicating the object identifier of the last-returned data.
+                        This object identifier is returned by calls to `BinaryDataByFilter` as the `last` value.
```

Comment:
Did you mean `TabularyDataByFilter`?

Reply:
Yes, yes I did haha thank you!!
```suggestion
                        This object identifier is returned by calls to `TabularDataByFilter` as the `last` value.
```